packages/hash/frontend/src/pages/[account-slug]/shared/white-card.tsx

```diff
@@ -43,6 +43,13 @@ export const WhiteCard = ({
           : {},
       ]}
     >
+      {/**
+       * @todo: refactor this to use `next/link` when a relative URL is passed
+       * into as the `href`, to avoid a flashing white screen when the user
+       * clicks on the entity's type.
+       *
+       * @see https://app.asana.com/0/1203179076056209/1203468350364504/f
+       */}
```

Comment:
@yusufkinatas not sure if you were planning to address this in existing work, but just wanted to flag that I've created [a task](https://app.asana.com/0/1203179076056209/1203468350364504/f) for this in the meantime.

Reply:
Thanks Ben, converted it to a subtask of "entity editor" task.